tests/testthat/test-convert_se_to_raw_data.R

```diff
@@ -28,6 +28,19 @@ test_that("convert_mae_to_raw_data works as expected with matrix data", {
   mae2 <- purrr::quietly(runDrugResponseProcessingPipeline)(
     input_df
   )
+  
+  
   test_synthetic_data(original, mae$result, data)
   test_synthetic_data(mae$result, mae2$result, data)
+  
+  trt <- BumpyMatrix::unsplitAsDataFrame(SummarizedExperiment::assay(original[[1]], "RawTreated"))
+  trt <- trt[!duplicated(trt$record_id), ]
+  df_raw1 <- convert_mae_to_raw_data(original)
+  SummarizedExperiment::assay(original[[1]], "RawTreated") <- BumpyMatrix::splitAsBumpyMatrix(
+    trt[!colnames(trt) %in% c("row", "column")],
+    row = trt$row,
+    column = trt$column)
+  df_raw2 <- convert_mae_to_raw_data(original)
```

Comment:
Why single-agent is completely removed here?

Reply:
Single-agent data still exists, only the swap of single agent is reverse and some drugs exists as a Gnumber_2 (to mimic the structure of internal GDS data)